source/developer/mobile-developer-setup.rst

```diff
@@ -66,24 +66,28 @@ Android
    Then reload your bash configuration: 
 
    ``source ~/.bash_profile``
-3. In the SDK Manager using Android Studio or the Android SDK command line tool, ensure the following are installed:
- - Android SDK Build-Tools (multiple versions)
-  - 23.0.3
-  - 25.0.3
-  - 26.0.1
- - Android Emulator
- - Android SDK Platform-Tools
- - Android SDK Tools
- - Android Support Library
- - Google Play services
- - Intel x86 Emulator Accelerator (HAXM installer)
- - Android Support Repository
- - Google Repository
- - Android API 23 (Marshmallow)
-  - Google APIs
-  - Android SDK Platform
-  - Google APIs Intel x86 Atom_64 System Image
- - Any other API version that you want to test
+3. In the SDK Manager using Android Studio or the Android SDK command line tool, ensure the following are installed (you may have to click "Show Package Details" to expand packages)
+ - SDK Tools
+  - .. image:: https://imgur.com/71wMoJj.png
+  - Android SDK Build-Tools (multiple versions)
+   - 23.0.3
+   - 25.0.3
+   - 26.0.1
+  - Android Emulator
+  - Android SDK Platform-Tools
+  - Android SDK Tools
+  - Google Play services
+  - Intel x86 Emulator Accelerator (HAXM installer)
+  - Support Repository
+   - Android Support Repository
+   - Google Repository
+ - SDK Platforms
+  - .. image:: https://imgur.com/Jm9RySe.png
```

Comment:
Do we want to host the images in imgur instead of in the repo? Every other image we have in the docs is included in the images folder, with my PR I actually added a sub directory for mobile

Reply:
Sure. I can upload them there. Another post I had seen used imgur, so I assumed that was best practice